<!doctype html>
<html class="no-js">
<head>
<title>Bennetzen Lab|Research</title>
<meta charset="utf-8">
<!--[if lt IE 9]><script src="js/html5.js"></script><![endif]-->
<link rel="stylesheet" media="all" href="css/style.css">
<meta name="viewport" content="width=device-width, initial-scale=1">
<!-- JS -->
<script src="js/jquery-1.7.1.min.js"></script>
<script src="js/custom.js"></script>
<script src="js/tabs.js"></script>
<script src="js/css3-mediaqueries.js"></script>
<script src="js/jquery.columnizer.min.js"></script>
<!-- Isotope -->
<script src="js/jquery.isotope.min.js"></script>
<!-- Lof slider -->
<script src="js/jquery.easing.js"></script>
<script src="js/lof-slider.js"></script>
<link rel="stylesheet" href="css/lof-slider.css" media="all" >
<!-- ENDS slider -->
<!-- Tweet -->
<link rel="stylesheet" href="css/jquery.tweet.css" media="all">
<!-- ENDS Tweet -->
<!-- superfish -->
<link rel="stylesheet" media="screen" href="css/superfish.css">
<script src="js/superfish-1.4.8/js/hoverIntent.js"></script>
<script src="js/superfish-1.4.8/js/superfish.js"></script>
<script src="js/superfish-1.4.8/js/supersubs.js"></script>
<!-- ENDS superfish -->
<!-- prettyPhoto -->
<script src="js/prettyPhoto/js/jquery.prettyPhoto.js"></script>
<link rel="stylesheet" href="js/prettyPhoto/css/prettyPhoto.css" media="screen">
<!-- ENDS prettyPhoto -->
<!-- poshytip -->
<link rel="stylesheet" href="js/poshytip-1.1/src/tip-twitter/tip-twitter.css" >
<link rel="stylesheet" href="js/poshytip-1.1/src/tip-yellowsimple/tip-yellowsimple.css" >
<script src="js/poshytip-1.1/src/jquery.poshytip.min.js"></script>
<!-- ENDS poshytip -->
<!-- JCarousel -->
<script type="text/javascript" src="js/jquery.jcarousel.min.js"></script>
<link rel="stylesheet" media="screen" href="css/carousel.css">
<!-- ENDS JCarousel -->
<!-- GOOGLE FONTS -->
<link href='http://fonts.googleapis.com/css?family=Voltaire' rel='stylesheet' type='text/css'>
<!-- modernizr -->
<script src="js/modernizr.js"></script>
<!-- SKIN -->
<link rel="stylesheet" media="all" href="css/skin.css">
<!-- Less framework -->
<link rel="stylesheet" media="all" href="css/lessframework.css">
<!-- jplayer -->
<link href="player-skin/jplayer-black-and-yellow.css" rel="stylesheet" type="text/css">
<script type="text/javascript" src="js/jquery.jplayer.min.js"></script>
<!-- flexslider -->
<link rel="stylesheet" href="css/flexslider.css" >
<script src="js/jquery.flexslider.js"></script>
</head>
<body class="blog">
<!-- HEADER -->
<header>
  <div class="wrapper cf">
    <div id="logo"> <a href="index.html"><img  src="img/logo.png" alt=""></a> </div>
    <!-- nav -->
    <ul id="nav" class="sf-menu">
      <li><a href="index.html">HOME</a></li>
       <li><a href="current.html">PEOPLE</a>
        <ul>
          <li><a href="current.html">CURRENT</a></li>
          <li><a href="alumni.html">ALUMNI</a></li>
        </ul>
      </li>
      <li><a href="publications.html">PUBLICATIONS</a></li>
      <li><a href="research.html">RESEARCH</a></li>
      <li><a href="covers.html">JOURNAL COVERS</a></li>
		<li><a href="recenthighlights.html">RECENT HIGHLIGHTS</a></li>
      <li><a href="gallery.html">GALLERY</a></li>
      <li><a href="contact.html">CONTACT</a></li>
    </ul>
    <div id="combo-holder"></div>
    <!-- ends nav -->
  </div>
</header>
<!-- ENDS HEADER -->
<!-- MAIN -->
<div id="main">
  <div class="wrapper cf">
    <!-- page content -->
    <div id="page-content" class="cf">
      <!-- Standard -->
      <article class="format-standard">
        <div class="feature-image" align="center"> <img src="Research imgs/Figure2F.jpg" align="center"> </div>
        <div class="accordion-trigger"> <h4 align="center">Plant genome structure and evolution, especially the nature of rearrangements <br>and the contributions of transposable elements</h4></div>
        <div class="accordion-container"> 
            <p>Molecular mapping using DNA probes, and now comprehensive DNA sequencing projects, have shown that gene content and order are significantly conserved between species, especially in closely related species. However, complex plant genomes like those in the angiosperms are relatively unstable, rearranging gene content and order at rates as much as 100-fold faster than mammalian genomes, for instance. We have begun to define the nature of these rearrangements and their mechanisms. Our preliminary data indicate that transposable elements are particularly clear sensors of these changes, many of which are caused by ectopic homologous recombination and illegitimate recombination. Ongoing and future studies will precisely determine the mechanisms of illegitimate recombination, will define the differences in the frequencies and types of these events in different plant lineages, and will measure how these processes contribute to the C value paradox. Bioinformatics and high throughput genomics are the key tools in these studies.</p>
          </div>
        </article>
        <p>
      <!-- ENDS  Standard -->
      <!-- Standard -->
      <article class="format-standard">
        <div class="feature-image" align="center"><img src="Research imgs/MaizeHap.jpg" alt="" width="417" height="240"></div>
        <div class="accordion-trigger"> <h4 align="center">The relationship between genome structure/evolution and gene function</h4></div>
        <div class="accordion-container"> 
            <p>We are only beginning to understand how genome context (that is, the nature of the DNA/chromatin that surrounds genes) can influence gene function and gene evolution. The grasses that we study are particularly well-suited to such endeavors because genome structure changes so rapidly. We are using the full set of genomics tools, including bioinformatics, full genome sequencing, expression analyses and reverse genetics, to try to determine the relationship between genomic context and gene expression. In addition, we are using these same tools to answer one of the great questions in biology. Namely, what are the specific genetic changes that differentiate species? In order to pursue these studies from a strong foundation, we need to have full genome sequence information for many species of plants. Current approaches to genome sequencing are too expensive to permit analysis of more than a few species, so we have developed gene-enrichment technologies that will allow low cost and very quick sequencing of any complex genome (plant, animal, fungal or protist). These technologies are now being tested in a project to sequence and orient the gene space within the maize nuclear genome. </p>
          </div>
      </article>
      <p>
      <!-- ENDS  Standard -->
      <!-- Standard -->
      <article class="format-standard">
        <div class="feature-image" align="center"> <img src="Research imgs/Girl.jpg" width="360" height="540"> </div>
          <div class="accordion-trigger"><h4 align="center">Genetic diversity and its use in under-utilized crops of the developing world</h4></div>
             <div class="accordion-container"> 
             <p>Modern crop improvement has led to the severe narrowing of germplasm diversity, both within species of crops and in the sets of crops that are now extensively utilized. Highly productive and stress-adapted cereals like tef (<span style="font-style:italic">Eragrostis tef</span>), finger millet (<span style="font-style:italic">Eleusine indica</span>) and foxtail millet (<span style="font-style:italic">Setaria italica</span>) are often replaced by less-adapted species like maize and wheat because of marketing forces. These "minor cereals" tend to be vital mainly to subsistence farmers in Africa and other parts of the developing world, so they have received little research attention. We are working on three subsistence cereals: tef, finger millet, and highland rice. Because they are so under-studied, the potential for significant improvement is much higher than for intensively studied/improved species like maize or lowland rice. Our efforts are mainly associated with studies of genetic diversity, and particularly any diversity that associates with useful agronomic properties like abiotic or biotic stress tolerance. A major component of this project involves the transfer of molecular technologies into developing world laboratories. </p>
        </div>
      </article>
      <p>
      <!-- ENDS  Standard -->
      <!-- Standard -->
      <article class="format-standard">
        <div class="feature-image" align="center"> <img src="Research imgs/project4photo.jpg" alt=""> </div>
        <div class="accordion-trigger"><h4 align="center">The rapid evolution of complex disease resistance loci in plants</h4></div>
            <div class="accordion-container">
            <p>Plants have many disease resistance genes that initiate a signal transduction cascade of antimicrobial factors after the pathogen signal is recognized. Many of these resistance genes are highly complex and highly unstable, often in ways that are comparable to events seen at the immunoglobin and histocompatibility genes of chordates. We are studying one of these genes, <span style="font-style:italic">Rp1</span> of maize, in significant detail. We have sequenced the entire complex of more than 650 kb. We are now characterizing gene expression, gene function, and the mechanisms of hyper-evolution in this cluster. In addition, we are studying and isolating other disease resistance genes from maize and sorghum. These new resistance genes will provide an interesting comparison to the <span style="font-style:italic">Rp1</span> complex and may also prove useful for crop improvement.</p>
        </div>
      </article>
      <p>
      <!-- ENDS  Standard -->
      <!-- Standard -->
      <article class="format-standard">
        <div class="feature-image" align="center"><img src="Slideshow imgs/Setaria italica.jpg" alt="" width="640" height="249"> </div>
        <div class="accordion-trigger"><h4 align="center">Models, mechanism and metagenomics for lignocellulosic ethanol production</h4></div>
            <div class="accordion-container">
            <p>Plants have evolved an efficient process, photosynthesis, for conversion of sunlight into usable energy. Most of the energy available for human use (coal, petroleum, and firewood) is derived from either ancient or current photosynthesis. Although the use of most biofuels is currently carbon negative or carbon neutral (that is, as much or more carbon is released into the atmosphere than is captured in the fuel), lignocellulosic energy conversion of perennial grasses like switchgrass has at least a seven-fold benefit in carbon capture, mostly by the roots and rhizosphere of the unharvested portions of the plants. We are involved in several consortia of projects that will attempt to improve the efficiency of lignocellulosic ethanol production so that it will be less expensive than current petroleum-based projects. The approach taken has three components. The first is development of foxtail millet (Setaria italica), an important grain of Northern China and Mongolia, as a model for switchgrass and other perennial grasses. We are collaborating with the Joint Genome Institute (DOE) in the sequencing and annotation of this genome. We are also working with other groups to improve the transformation, genetic maps and other components of the genetic/genomic toolkit for this species. Second, we are cloning and characterization genes from switchgrass and foxtail millet that are known to be involved in both the quality and quantity of biomass produced by these species. Third, and finally, we are employing metagenomic analysis of the microbial components of the switchgrass rhizosphere to find the microbes that will help sustain and promote high biomass quality and yields in this perennial crop. </p>
          </div>
      </article>
      <p>
      <!-- ENDS  Standard -->
    </div>
    <!-- ENDS posts list -->
  </div>
  <!-- ENDS WRAPPER -->
</div>
<!-- ENDS MAIN -->
<!-- FOOTER -->
<footer>
  <div class="wrapper cf">
    <!-- widgets -->
    <!-- bottom -->
    <div class="footer-bottom">
      <div class="left"> Copyright Â© 2015 Bennetzen Lab. Template by <a href="http://www.luiszuno.com/">luiszuno.</a>
      </div>
    </div>
    <!-- ENDS bottom -->
  </div>
</footer>
<!-- ENDS FOOTER -->
</body>
</html>
